### PR TITLE
Added support for CMS 3.7 and Django 2.2

### PR DESCRIPTION
**Pull request by bplociennik**

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -3,9 +3,12 @@ Changelog
...
-1.3.2 (unreleased)
+1.4.0 (unreleased)
 ==================
 
+* Introduced support for CMS 3.7 and Django 2.2
+* Removed support for Django 2.0
+* Extended test matrix
 * Added isort and adapted imports
 * Adapted code base to align with other supported addons
 * Added option to add custom classes to carousel (#82)
```
Comment (chaosk) [inline CHANGELOG.rst:14]: Could you release these fixes as another patch release? It might affect people using 2.0, for example.

**Review by chaosk — CHANGES_REQUESTED** [review]

File: CHANGELOG.rst (hunk 1 above)
Comment (chaosk) [reply CHANGELOG.rst:14]: BTW 1.3.1 changelog entry has a typo: it should be 201**8**, not 2012

Comment (FinalAngel) [reply CHANGELOG.rst:14]: @chaosk I'd like to release this as one release, it has already breaking changes and Django 2.0 is not supported anymore. Do you actively use it?

Comment (chaosk) [reply CHANGELOG.rst:14]: Fair enough, although I don't use 2.0, but that doesn't mean there aren't people who do.

**Review by chaosk — APPROVED** [review]

**Pull request merged**